lib/animation-frame.js

```diff
@@ -5,9 +5,8 @@ var now = require('./now')
 var performance = require('./performance')
 var root = require('./root')
 
-// Weird native implementation doesn't work if context is defined.
```

Comment:
that comment says that this is exactly what you should not be doing, because this was failing in some browsers

Reply:
not sure what browser you are testing for, also, is there still a need for this library? it is supposed to be useful for very very old browsers!

Reply:
@kof yea, you are right. we don't need this library. thx